R/data.R

```diff
@@ -0,0 +1,78 @@
+#' Synthetic dataset of stochastic SIR system with known Rt
+#'
+#' A dataset produced by Katelyn Gostic as part of a published manuscript:
+#' Gostic, Katelyn M., et al. "Practical considerations for measuring the
+#' effective reproductive number, R t." PLoS computational biology 16.12 (2020):
+#' e1008409. The data are simulated from a stochastic SEIR compartment model.
+#'
+#' This synthetic dataset has a number of desirable properties: 1. The force of
+#' infection is manipulated by changing the Rt, allowing for sudden changes in
```

Comment:
```suggestion
#' infection changes depending on the Rt, allowing for sudden changes in
```

I don't like "manipulated" (too complicated word and bad associations).